doc/man7/EVP_KDF-PBKDF2.pod

```diff
@@ -57,15 +57,12 @@ The checks performed are:
 
 The default provider uses a default mode of 1 for backwards compatibility,
 and the FIPS provider uses a default mode of 0.
-
-The value string is expected to be a decimal number 0 or 1.
-Setting this to zero will ignore the error and set the approved "fips-indicator"
-to 0.
+Setting this value to 1 in the OpenSSL FIPS provider, breaks FIPS compliance.
```

Comment:
Does it break the compliance unconditionally? If the indicator returns "approved", then no compliance should be broken.

Reply:
Ok..
Had to change it everywhere (Hope the wording is better now)